.github/workflows/integration-test-src.yml

```diff
@@ -1,6 +1,12 @@
 name: INTEGRATION-TEST-SRC
 
 on:
+  push:
+    branches:
+      - main
+  pull_request:
```

Comment:
both trigger on push and pull_request will take so much resource. I suggest only on `schedlue`

Reply:
Change to only schedlue